src/kernel/scheduler.zig

```diff
@@ -129,17 +139,19 @@ pub fn init(allocator: *Allocator) Allocator.Error!void {
     errdefer allocator.destroy(current_task);
     // PID 0
     current_task.pid = 0;
-    current_task.stack = @intToPtr([*]u32, @ptrToInt(&KERNEL_STACK_START))[0..4096];
+    current_task.kernel_stack = @intToPtr([*]u32, @ptrToInt(&KERNEL_STACK_START))[0..4096];
```

Comment:
Could try and use `KERNEL_STACK_END` in case we change the stack size later for the length.